docs/eos-learn/migration-to-mandel.md

```diff
@@ -0,0 +1,41 @@
+---
+id: migration-to-mandel
+title: Migration from EOSIO to Mandel
+sidebar_label: Migration from EOSIO to Mandel
+---
+
+## Dissatisfaction with Block One
+
+In 2021, the EOS community felt disappointed about the compromise of [Block One](https://block.one/) with the [EOS Network Foundation](https://eosnetwork.com/) (ENF) because use case proposals were missing, and the little support in the blockchain. For these reasons, through a super-majority consensus of Block Producers, the community affirms that the company doesn't act in the interest of the community and were stop the payment that the company receives for the network maintenance.
```

Comment:
change compromise for commitment

Reply:
The [EOS Network Foundation](https://eosnetwork.com/) (ENF) emerged as a new alternative to direct network funds into furthering the development of the EOSIO protocol and EOS mainnet.